pages/wi/partner-listing.js

```diff
@@ -0,0 +1,189 @@
+import React from 'react';
```

Comment:
Instead of `partner-listing` what do you think about just calling this `partners`?

Reply:
makes sense...changed